core/client-native/src/main/java/com/thousandeyes/sdk/client/NativeApiClient.java

```diff
@@ -123,7 +123,7 @@ private <T> ApiResponse<T> processResponse(HttpResponse<InputStream> response,
             return new ApiResponse<>(
                     response.statusCode(),
                     response.headers().map(),
-                    body == null ? null : reader.readValue(body)
+                    reader.getValueType().hasRawClass(Void.class) ? null : reader.readValue(body)
```

Comment:
The two operations provided in the ticket, `delete credential` and `run instant test` didn't have a null `body` and it differed between one another. I wish I could have found an explanation for this but I didn't and because of this I couldn't even use one of `body`'s built in methods.

I also couldn't use the status code because one is a `204` and the other is a `200`. All I was left with was the return type, that was common for both